test/framework/src/main/java/org/elasticsearch/test/AbstractWireSerializingTestCase.java

```diff
@@ -25,11 +25,15 @@ public abstract class AbstractWireSerializingTestCase<T extends Writeable> exten
      */
     protected abstract Writeable.Reader<T> instanceReader();
 
+    protected Writeable.Writer<T> instanceWriter() {
```

Comment:
It's probably worth javadoc on this one too. There re like 1000 subclasses of this thing.